src/frontend/js/components/SaleTunnelStepPayment/index.tsx

```diff
@@ -0,0 +1,275 @@
+   *
+   * @returns {Joanie.Address[]} Sorted addresses ascending by title
+   */
+  const addressesItems = useMemo(() => {
+    const items = [...addresses.items];
+
+    // Add local address to the address item list.
+    if (selectedAddress?.id === LOCAL_BILLING_ADDRESS_ID) {
+      items.push(selectedAddress);
+    }
+
+    return items.sort(sortAddressByTitleAsc);
+  }, [addresses.items, selectedAddress]);
+
+  /**
+   * Retrieve address through its `id` provided by the event target value
+   * then update `selectedAddress` state
+   *
+   * @param {React.ChangeEvent<HTMLSelectElement>} event
+   */
+  const handleSelectAddress = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const targetedAddress = addresses.items.find((a) => a.id === event.target.value);
+    if (targetedAddress) {
+      setAddress(targetedAddress);
+    }
+  };
+
+  const toggleCreditCard = (creditCardId: string) => {
+    if (selectedCreditCard === creditCardId) {
+      setCreditCard(null);
+    } else {
+      setCreditCard(creditCardId);
+    }
+  };
+
+  if (showAddressCreationForm) {
+    return (
+      <AddressesManagement
+        handleClose={() => setShowAddressCreationForm(false)}
+        selectAddress={setAddress}
+      />
+    );
+  }
+
+  if (!user) return null;
+
+  return (
+    <section className="SaleTunnelStepPayment">
+      <section className="SaleTunnelStepPayment__block">
+        <header className="SaleTunnelStepPayment__block__header">
+          <h5 className="SaleTunnelStepPayment__block__title">
+            <FormattedMessage {...messages.resumeTile} />
+          </h5>
+        </header>
+        <div className="SaleTunnelStepPayment__block--product">
+          <h6 className="SaleTunnelStepPayment__block--product__title">{product.title}</h6>
+          <p className="SaleTunnelStepPayment__block--product__price">
+            <FormattedNumber
+              value={product.price}
+              style="currency"
+              currency={product.price_currency}
+            />
+          </p>
+        </div>
+      </section>
+      <section className="SaleTunnelStepPayment__block">
+        <header className="SaleTunnelStepPayment__block__header">
+          <h5 className="SaleTunnelStepPayment__block__title">
+            <FormattedMessage {...messages.userTile} />
+          </h5>
+        </header>
+        <div className="SaleTunnelStepPayment__block--buyer">
+          <h6 className="SaleTunnelStepPayment__block--buyer__name">
+            {user.fullname || user.username}
+          </h6>
+          {user.email ? (
+            <p className="SaleTunnelStepPayment__block--buyer__email">{user.email}</p>
+          ) : null}
+          <header className="SaleTunnelStepPayment__block--buyer__address-header">
+            <h6>
+              <FormattedMessage {...messages.userBillingAddressFieldset} />
+            </h6>
+            {addressesItems.length > 0 && (
+              <button
+                aria-hidden="true"
```

Comment:
Watch out for this, I guess the aria-hidden was meant on the icon inside the button?

Reply:
Oh yes 🤦‍♂️